Content.Shared/ReagentOnItem/Systems/SpaceLubeOnItemSystem.cs

```diff
@@ -0,0 +1,114 @@
+using Content.Shared.IdentityManagement;
+using Content.Shared.Popups;
+using Content.Shared.Throwing;
+using Robust.Shared.Random;
+using Content.Shared.Clothing.Components;
+using Content.Shared.Inventory;
+using Content.Shared.Examine;
+using Robust.Shared.GameStates;
+using Content.Shared.Hands;
+using Content.Shared.NameModifier.EntitySystems;
+
+namespace Content.Shared.ReagentOnItem;
+
+public sealed class SpaceLubeOnItemSystem : EntitySystem
+{
+    [Dependency] private readonly ThrowingSystem _throwing = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+    [Dependency] private readonly InventorySystem _inventory = default!;
+    [Dependency] private readonly SharedTransformSystem _xform = default!;
+    [Dependency] private readonly NameModifierSystem _nameMod = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, ComponentInit>(OnInit);
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, GotAttemptedHandPickupEvent>(OnHandPickUp);
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, ExaminedEvent>(OnExamine);
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, RefreshNameModifiersEvent>(OnRefreshNameModifiers);
+
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, ComponentGetState>(GetSpaceLubeState);
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, ComponentHandleState>(HandleSpaceLubeState);
+    }
+
+    private void OnInit(EntityUid uid, SpaceLubeOnItemComponent component, ComponentInit args)
+    {
+        _nameMod.RefreshNameModifiers(uid);
+    }
+
```

Comment:
This is going to break save/load, are you sure it isn't supposed to be mapinit.

Reply:
Yep your right!